orc8r/cloud/go/services/metricsd/prometheus/prometheus-cache/cache/cache_test.go

```diff
@@ -224,7 +248,68 @@ mf1 123 1
 mf1 234 2
 mf1 456 3
 `
-	assert.Equal(t, expectedExpositionText, cache.exposeMetrics(cache.metricFamiliesByName))
+	assert.Equal(t, expectedExpositionText, cache.exposeMetrics(cache.metricFamiliesByName, 1))
+}
+
+func assertWorkerPoolHandlesError(t *testing.T) {
```

Comment:
test cases should be exported functions

Reply:
I have an exported method TestScrapeBadMetrics above that calls this method. Should I copy this code into that method?